Backpack-SwiftUI/FlowStackView/Classes/BPKFlowStackView.swift

```diff
@@ -75,22 +75,33 @@ public struct BPKFlowStackView<Data: Collection, Content: View>: View where Data
         }
     }
     
-    func computeRows() -> [[Data.Element]] {
-        var rows: [[Data.Element]] = [[]]
+    private struct Item: Hashable, Identifiable {
+        let id: Int
+        var data: Data.Element
+    }
+    
+    private func computeRows() -> [[Item]] {
+        var rows: [[Item]] = []
         var currentRow = 0
         var remainingWidth = availableWidth
+        var currentIndex = 0
         
         data.forEach { element in
-            let elementSize = elementsSize[element, default: CGSize(width: availableWidth, height: 1)]
+            let defaultSize = CGSize(width: availableWidth, height: 1)
+            let elementSize = elementsSize[element, default: defaultSize]
+            let fitsInCurrentRow = elementSize.width + spacing.width > remainingWidth
             
-            if elementSize.width + spacing.width > remainingWidth {
+            if fitsInCurrentRow { // new row
                 remainingWidth = availableWidth
                 currentRow += 1
-                rows.append([element])
-            } else {
-                rows[currentRow].append(element)
+                rows.append([Item(id: currentIndex, data: element)])
```

Comment:
if it fits in the current row, we create a new row? Is the if-statement correct?

Reply:
(I believe this is a variable naming issue)

Reply:
ah yes, it's a naming issue! i'll rename, it's incorrectly named now